playground/frontend/lib/modules/editor/components/share_dropdown/share_tabs_headers.dart

```diff
@@ -0,0 +1,68 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import 'package:flutter/material.dart';
+import 'package:playground/constants/sizes.dart';
+import 'package:playground/pages/playground/states/playground_state.dart';
+import 'package:provider/provider.dart';
+import 'package:flutter_gen/gen_l10n/app_localizations.dart';
+
+const kTabBarWidth = 180.0;
```

Comment:
This should not be hardcoded. The natural text width should dictate the widgets' width.

Reply:
When I remove this const, I get this error: 
Error: Assertion failed: file:///.../flutter/packages/flutter/lib/src/rendering/box.dart:1979:12
hasSize
"RenderBox was not laid out: RenderFractionalTranslation#8585f relayoutBoundary=up1"

So I guess I'll keep it.

Reply:
This is piling of tech debt. I can easily see a third tab like 'social' to post the snippet on social media at some point.

The investigation now will take some effort. But if this is not attained, then even more effort will be needed by some guys adding a 3rd tab in the future or localizing the app with longer or shorter strings.